mlflow/server/js/src/components/ParallelCoordinatesPlotControls.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TreeSelect } from 'antd';
+
+export class ParallelCoordinatesPlotControls extends React.Component {
+  static propTypes = {
+    paramKeys: PropTypes.arrayOf(String).isRequired,
```

Comment:
nit: could you add a small comment just saying that this is the set of param keys which are selectable?

Reply:
Will add.